thesis-api-v2/src/main/java/pl/edu/agh/quizzesthesis/api/dto/QuestionsSetKeyRequest.java

```diff
@@ -0,0 +1,3 @@
+package pl.edu.agh.quizzesthesis.api.dto;
+
+public record QuestionsSetKeyRequest(String questionsSetKey) {}
```

Comment:
I am not sure if creating a new object (record) for just simple `String` makes any sense...

Reply:
IMO it's perfectly fine.